tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_IMAP_email_notification.py

```diff
@@ -60,8 +60,11 @@ def test_IMAP_email_notification(self):
 
         # Assert there is one notification and is listed in notifications-container
         notifications = utility_tray.notifications
-        self.assertEqual(1, len(notifications), 'Expected one notification.')
-        email = notifications[0].tap_notification()
```

Comment:
Actually, this might still used because notifications[0] must have come from the email app in this case. I think I'll revert this change, unless you don't agree.